frontend/packages/console-telemetry-plugin/src/flags/detect-telemetry.ts

```diff
@@ -0,0 +1,6 @@
+import { SetFeatureFlag } from '@console/dynamic-plugin-sdk';
+
+export const detectTelemetry = (setFeatureFlag: SetFeatureFlag) => {
+  const { disabled: isTelemetryDisabled = false } = window.SERVER_FLAGS.telemetry;
+  setFeatureFlag('TELEMETRY', !isTelemetryDisabled);
```

Comment:
`window.SERVER_FLAGS.telemetry` flags should be defined as `Record<string, string>`. We should define a type for this somewhere.

```suggestion
  const { disabled } = window.SERVER_FLAGS.telemetry;
  setFeatureFlag('TELEMETRY', disabled !== 'true');
```